src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -131,7 +131,15 @@
 Get_PublicKey_Info_Fault_Type = 'Error while fetching the PoP publickey information from client proxy'
 PoP_Public_Key_Expried_Fault_Type = 'The PoP public key used to generate the at has expired'
 Post_AT_To_ClientProxy_Failed_Fault_Type = 'Failed to post access token to client proxy'
-Kubectl_Get_Events_Failed_Fault_Type = "Error while doing kubectl get events"
+Kubectl_Get_Events_Failed_Fault_Type = "Error occured while doing kubectl get events"
+Kubectl_Get_Secrets_Failed_Fault_Type = "Error occured while doing kubectl get secrets"
+Kubectl_Get_Secrets_Failed_Fault_Type_Exception = "Exception occured while doing kubectl get secrets"
+Helm_Values_Save_Failed_Error = "Error occured while doing helm get values for azure-arc release"
+Helm_Values_Save_Failed_Exception = "Exception occured while doing helm get values for azure-arc release"
+Metadata_CR_Save_Failed_Error = "Error occured while fetching metadata CR snapshot"
+Metadata_CR_Save_Failed_Exception = "Exception occured while fetching metadata CR snapshot"
```

Comment:
Let's end the name with "fault_type" instead of exception or error for consistency